pkgs/borgbackup/borg/registry.yaml

```diff
@@ -34,12 +34,12 @@ packages:
           - linux/amd64
           - darwin
       - version_constraint: "true"
-        asset: borg-{{.OS}}64
+        asset: borg-{{.OS}}
         format: raw
         rosetta2: true
         replacements:
-          amd64: x64
-          darwin: macos
+          linux: linux-glibc231
```

Comment:
I am not familiar with aquaproj or its usages / users, but that means it will likely require glibc >= 2.31 on the target system.

Reply:
✅ Changed to the 2.28